lib/imagemounter/index.js

```diff
@@ -0,0 +1,119 @@
+import { readFile, existsSync, lstatSync, createReadStream } from 'fs';
```

Comment:
please use async methods where possible. calling of sync ones blocks the event loop and might have negative influence to the whole server functionality

Reply:
I prefer to use async methods on heavy operations like `readFile`. Will it be really matter to use `existsSync` or `lstatSync`?

Reply:
yes. We never know how slow IO could be

Reply:
> yes. We never know how slow IO could be

Alright, I will rewrite them using `fs/promises`